ee/modules/650-runtime-audit-engine/docs/EXAMPLES_RU.md

```diff
@@ -79,3 +79,10 @@ spec:
         (command=%proc.cmdline pid=%proc.pid connection=%fd.name sport=%fd.sport user=%user.name %container.info image=%container.image)
       priority: Notice
 ```
+
```

Comment:
add `Notifying when a shell is run in a container` example

Reply:
done